### ISSUE TITLE
OMG :(

### ISSUE THREAD

**Issue by inya007** [sbaril/Photoshop-Animation#13]
Hi!
Is there any link that Github newbie can use? I can not even find the link to ZXP file of AnimDessin ... The addon was retracted from Adobe and I am obviously incapable of understanding Github. Please. Please. Thanks for reading, M

**Comment by plindale** [comment]
https://github.com/sbaril/Photoshop-Animation

**Comment by schroef** [comment]
@plindale what use is sending a link to here? We are already in that repo?!

**Comment by schroef** [comment]
@inya007 in this repo you can find the zxp as well. You can use manual install method and see if that works. Otherwise there is a manual method, but that is more work.

Let us know if you did get the zxp installed and working

**Comment by inya007** [comment]
Yes, I found Stefanes youtube channel with explanation. Thanks a lot Stefane!
If anybody is looking: https://www.youtube.com/channel/UCorsNUHqIeH7-6L_eg1iMSQ
